question_service/config/question_data.js

```diff
@@ -0,0 +1,281 @@
+const questions = [
+    {
+      title: 'Reverse a string',
+      description: `
+      Write a function that reverses a string. The input string is given as an array
+      of characters s.
+      You must do this by modifying the input array in-place with O(1) extra
+      memory.
+      Example 1:
+      Input: s = ["h","e","l","l","o"]
+      Output: ["o","l","l","e","h"]
+      Example 2:
+      Input: s = ["H","a","n","n","a","h"]
+      Output: ["h","a","n","n","a","H"]
+      Constraints:
+      - 1 <= s.length <= 105
+      - s[i] is a printable ascii character.
+      `,
+      category: ['String', 'Algorithms'],
+      complexity: 'Easy',
```

Comment:
sorry, can you change all the field names for `category` to `topics` and `complexity` to `difficulty`